app/src/androidTest/java/org/mozilla/tv/firefox/helpers/MainActivityTestRule.kt

```diff
@@ -34,7 +34,7 @@ class MainActivityTestRule(
      * implementation changes and the tests break. In that case, this code might be
      * broken because it's not used, and thus tested, at present.
      */
-    private lateinit var loadingIdlingResource: SessionLoadedIdlingResource
+    lateinit var loadingIdlingResource: SessionLoadedIdlingResource
```

Comment:
`@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)`? :)

Reply:
TBH, I would argue that we don't need the annotation at all.  This file is in the `androidTest` directory, so it isn't available from anywhere but in our tests.